litestar/cli/_utils.py

```diff
@@ -10,15 +10,25 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Any, Callable, Generator, Iterable, Sequence, TypeVar, cast
 
-from click import ClickException, Command, Context, Group, pass_context, style
-from rich.console import Console
+from rich import get_console
 from rich.table import Table
 from typing_extensions import Concatenate, ParamSpec
 
 from litestar import Litestar, __version__
 from litestar.middleware import DefineMiddleware
 from litestar.utils import get_name
 
+try:
+    from rich_click import ClickException, Context, pass_context
+    from rich_click.rich_command import RichCommand as Command
+    from rich_click.rich_group import RichGroup as Group
+
+    rich_click_installed = True  # pragma: no cover
+except ImportError:
+    from click import ClickException, Command, Context, Group, pass_context  # type: ignore[assignment]
+
+    rich_click_installed = False
+
```

Comment:
Is this actually needed? You added `rich-click` to the `cli` extra, so it will always be installed when the other CLI dependencies are installed.